src/main/java/com/iexec/sms/tee/config/SconeInternalServicesConfiguration.java

```diff
@@ -0,0 +1,33 @@
+package com.iexec.sms.tee.config;
+
+import com.iexec.common.tee.TeeEnclaveProvider;
+import com.iexec.sms.api.config.SconeServicesProperties;
+import com.iexec.sms.api.config.TeeAppProperties;
+import com.iexec.sms.api.config.TeeServicesProperties;
+import com.iexec.sms.tee.ConditionalOnTeeProvider;
+import lombok.Getter;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+import javax.validation.constraints.NotBlank;
+
+@Configuration
+@ConditionalOnTeeProvider(providers = TeeEnclaveProvider.SCONE)
+@Getter
+public class SconeInternalServicesConfiguration
```

Comment:
I'm a bit puzzled with that one .. :)
```
public class SconeInternalServicesConfiguration
        extends SconeServicesProperties
        implements TeeInternalServicesConfiguration
```

Reply:
So was I. I remember I had to introduce this interface for some reason, but it has suffered so much modifications it doesn't seem to be useful anymore. Please see https://github.com/iExecBlockchainComputing/iexec-sms/pull/119/commits/dc03cf0fca9a74168ca02549995730050d857aa6.